packages/theme/src/theme/_navs.scss

```diff
@@ -87,10 +88,10 @@
 				color: rgba(1, 1, 1, 0.6);
 				background: transparent;
 				border: none;
-				outline: none;
 
 				&:focus {
 					background: rgba(0, 0, 0, 0.06);
+					outline: none;
```

Comment:
This introduce accessibility issue. When navigating with keyboard we have no clue where the focus is.